test/unit/setup.spec.js

```diff
@@ -0,0 +1,138 @@
+import { expect } from 'chai'
+import setup from '../../src/setup'
+
+describe('unit tests for setup.js', () => {
+  it('should use default settings', async () => {
+    const settings = setup({})
+    expect(settings.apiVersion).to.be.equal(2)
+    expect(settings.apiUrl).to.be.equal('https://api.us.evrythng.io/v2')
+    expect(settings.region).to.be.equal('us')
+  })
+  it('should use custom apiUrl', async () => {
+    const settings = setup({ apiUrl: 'google.com' })
+
+    expect(settings.apiVersion).to.be.equal(2)
+    expect(settings.apiUrl).to.be.equal('google.com')
+    expect(settings.region).to.be.equal('us')
+  })
+  it('should use custom apiUrl even if apiVersion is setup', async () => {
+    const settings = setup({ apiVersion: 1, apiUrl: 'google.com' })
+    expect(settings.apiVersion).to.be.equal(1)
+    expect(settings.apiUrl).to.be.equal('google.com')
+    expect(settings.region).to.be.equal('us')
+  })
+  it('should use custom apiUrl even if apiVersion and region are setup', async () => {
+    const settings = setup({ apiVersion: 2, region: 'eu', apiUrl: 'google.com' })
+    expect(settings.apiVersion).to.be.equal(2)
+    expect(settings.apiUrl).to.be.equal('google.com')
+    expect(settings.region).to.be.equal('eu')
+  })
+  it('should set url based on apiVersion:2  and region:eu', async () => {
```

Comment:
Suggest either `URL` or `apiUrl` instead of just `url` for these `it`s

Reply:
fixed, thanks